src/platform-implementation-js/dom-driver/gmail/views/gmail-compose-view/add-button.ts

```diff
@@ -19,6 +28,7 @@ export default function addButton(
   >,
   groupOrderHint: string,
   extraOnClickOptions: Record<string, any>,
+  bus: Bus<AddedButtonEvents, unknown>,
 ) {
   return new Promise<Options | null>((resolve) => {
```

Comment:
We should probably remove the wrapping Promise here to account for the new `Bus` API.